Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -382,3 +366,102 @@ void GeneralPane::GenerateNewIdentity()
+#ifdef USE_DISCORD_PRESENCE
+  static constexpr char TR_DISCORD_PRESENCE_DESCRIPTION[] =
+      QT_TR_NOOP("Shows which game is active in your Discord status and the duration of your "
+                 "current play session."
+                 "<br><br>This setting cannot be changed while emulation is active."
+                 "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+#endif
+  static constexpr char TR_SPEEDLIMIT_DESCRIPTION[] =
+      QT_TR_NOOP("Sets the maximum speed at which games will run. If your computer is not fast "
+                 "enough the actual speed may be slower."
+                 "<br><br><dolphin_emphasis>If unsure, select 100%.</dolphin_emphasis>");
+  static constexpr char TR_UPDATE_TRACK_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects which update track Dolphin uses when checking for updates at startup. If a new "
+      "update is available Dolphin will show a list of changes made since your current version and "
+      "ask you if you want to update."
+      "<br><br>The Dev track has the latest version of Dolphin which often updates "
+      "multiple times per day. Select this track if you want the newest features and "
+      "fixes."
+      "<br><br>The Beta track has an update every 1-3 months. Some reasons you might "
+      "prefer to use this track:"
+      "<br>- You prefer using versions that have additional testing before release."
+      "<br>- NetPlay requires players to have the same Dolphin version, and the latest "
+      "Beta version will have the most players to match with."
+      "<br>- You frequently use Dolphin's Save State system, which doesn't guarantee "
+      "backward compatibility of Save States between Dolphin versions. If this applies "
+      "to you make sure you make an in-game save before updating (i.e. save your game "
+      "in the same way you would on a physical GameCube or Wii), then load the in-game "
+      "save after updating Dolphin and before making any new Save States."
+      "<br><br>Selecting \"Don't Update\" will prevent Dolphin from automatically checking for "
+      "updates."
+      "<br><br><dolphin_emphasis>If unsure, select Beta.</dolphin_emphasis>");
+  static constexpr char TR_FALLBACK_REGION_DESCRIPTION[] =
+      QT_TR_NOOP("Sets the region used for titles whose region cannot be determined automatically."
+                 "<br><br>This setting cannot be changed while emulation is active.");
+  static constexpr char TR_ENABLE_ANALYTICS_DESCRIPTION[] = QT_TR_NOOP(
+      "If selected, Dolphin can collect data on its performance, feature usage, and configuration, "
+      "as well as data on your system's hardware and operating system. <br><br>No private data is "
+      "ever collected. This data helps us understand how people and emulated games use Dolphin and "
+      "prioritize our efforts. It also helps us identify rare configurations that are causing "
+      "bugs, performance and stability issues.");
```

Comment:
It might be worth stating whether information on games you boot with Dolphin is being sent with this setting (and if so, to what extent).